os/board/rtl8730e/src/component/wifi/driver/include/rtw_wifi_constants.h

```diff
@@ -957,6 +957,115 @@ enum gen_ie_type {
 	P2PWPS_ASSOC_RSP_IE
 };
 
+#if defined(CONFIG_PLATFORM_TIZENRT_OS)
+enum wpa_states {
```

Comment:
This was added as I couldn't find these definitions in TizenRT. Are there plans to add these in TizenRT? I would like to remove this if TizenRT have plans to include these in their header files

Reply:
We will not include this in TizenRT wifi. We would like to use this value from Realtek as just an interger value. We may use include this in the header files as a comment. If you can keep this here, it would be helpful.

Reply:
Ok, will keep it in this header file